### PR TITLE
samples: Aligned the radio_test TX power with FEM

### PR DESCRIPTION
**Pull request by dchat-nordic**
Corrected the behavior of the radio_test sample when used with FEM.

Jira: NCSDK-23709

### CODE REVIEW

File: doc/nrf/releases_and_maturity/releases/release-notes-2.5.0.rst [hunk 1]
```diff
@@ -804,6 +804,7 @@ Other samples
 * :ref:`radio_test` sample:
 
   * Updated the sample by aligning the timer's configuration to the new nrfx API.
+  * Corrected setting the TX power with FEM.
```
Comment (peknis) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.5.0.rst:807]:
```suggestion
  * Corrected the way of setting the TX power with FEM.
```
BTW, why in release notes 2.5.0 and not in the changelog?

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-2.5.0.rst (hunk 1 above)
Comment (dchat-nordic) [reply doc/nrf/releases_and_maturity/releases/release-notes-2.5.0.rst]: This was a mistake when selecting file from list.

**Review by peknis — APPROVED** [review]

File: samples/peripheral/radio_test/src/radio_test.c [hunk 2]
```diff
@@ -91,6 +91,92 @@ static uint16_t channel_to_frequency(nrf_radio_mode_t mode, uint8_t channel)
...
+
+#if defined(RADIO_TXPOWER_TXPOWER_Neg7dBm)
+	case -7:
+		return RADIO_TXPOWER_TXPOWER_Neg7dBm;
+#endif /* defined(RADIO_TXPOWER_TXPOWER_Neg7dBm) */
+#if defined(RADIO_TXPOWER_TXPOWER_Neg6dBm)
+	case -6:
+		return RADIO_TXPOWER_TXPOWER_Neg6dBm;
+#endif /* defined(RADIO_TXPOWER_TXPOWER_Neg6dBm) */
+#if defined(RADIO_TXPOWER_TXPOWER_Neg5dBm)
```
Comment (maje-emb) [inline samples/peripheral/radio_test/src/radio_test.c:123]:
```suggestion
#endif /* defined(RADIO_TXPOWER_TXPOWER_Neg7dBm) */

#if defined(RADIO_TXPOWER_TXPOWER_Neg6dBm)
	case -6:
		return RADIO_TXPOWER_TXPOWER_Neg6dBm;
#endif /* defined(RADIO_TXPOWER_TXPOWER_Neg6dBm) */

#if defined(RADIO_TXPOWER_TXPOWER_Neg5dBm)
```

Similarly in other places.

**Review by maje-emb — APPROVED** [review]

File: samples/peripheral/radio_test/src/radio_test.c (hunk 2 above)
Comment (dchat-nordic) [reply samples/peripheral/radio_test/src/radio_test.c]: Corrected

**Review by doki-nordic — APPROVED** [review]

**Pull request merged**